### DISCUSSION TITLE
Downloading a bunch of tabs open to ao3 in a window

### DISCUSSION THREAD

**Discussion by emcneil9114** [nianeyna/ao3downloader#175] [Q&A]
Hi! I don't know if it is already a function in the way somewhere, but is it possible to use this code to download fics from a browser when you have multiple tabs opened to AO3, without having to copy paste individually? Like an option for download all open tabs from AO3 on the insert specific browser. 

On any given day, I have at least 100 tabs open to A03 that don't always make it to the bookmarks folder or eventually lose when my browser crashes.

Thanks!

**Comment by nianeyna** [comment]
Not directly, but if you can find an extension or some other method to get a plaintext list of your open ao3 tabs you should be able to feed that into the downloader
